frontend-react/src/pages/admin/value-set-editor/ValueSetsIndex.tsx

```diff
@@ -110,7 +84,7 @@ const ValueSetsIndex = () => {
                 <title>Value Sets | Admin | {PAGE_TITLE}</title>
             </Helmet>
             <section className="grid-container">
-                <ValueSetsTable />
+                {withNetworkCall(<ValueSetsTable />)}
```

Comment:
[for later] since this replaces the whole page with a spinner, I think we need to refactor our table component to allow for the static frame of the table display, and just wrap the dynamic piece in the suspense. This is still a big improvement!

Reply:
Interesting -- I like the idea. I think we'd have to find a way to pass the network hook in to Table -> TableRows, and then wrap it as `{withNetworkCall(<TableRows {...propsThatIncludeNetworkCall} />)}`

Reply:
You do a similar hook pass with our session context and okta's hook!

Reply:
#6735